pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -1287,6 +1287,23 @@ def test_opening_time(self, case):
         datetime(2014, 7, 7, 19, 30): datetime(2014, 7, 5, 4, 30),
         datetime(2014, 7, 7, 19, 30, 30): datetime(2014, 7, 5, 4, 30, 30)}))
 
+    # long business hours
```

Comment:
Reference this PR as part of this comment e.g. add:

`see gh-26381`

To the end of that comment.

Reply:
Reference done